### ISSUE TITLE
Getting errors running tests in Meteor 1.3-rc.2

### ISSUE THREAD

**Issue by jakobrosenberg** [deanius/meteor-promise#22]
Just upgraded from ``1.3-beta.12`` to ``1.3-rc.2`` and now I get the following error

```
 While processing files with cosmos:browserify (for target web.browser):
   packages/deanius:promise/client/runAsync.browserify.js: Cannot read
   property '0' of undefined
```

EDIT: The error is present outside of testing as well.

**Comment by deanius** [comment]
Maintainers and pull requests welcome!

**Comment by jakobrosenberg** [comment]
This was resolved in Meteor 1.3, so I'm closing this one.

**Comment by deanius** [comment]
Thanks for the update.
